guides/source/active_record_querying.md

```diff
@@ -493,38 +557,39 @@ NOTE: Only equality, range, and subset checking are possible with Hash condition
 #### Equality Conditions
 
 ```ruby
-Client.where(locked: true)
+Book.where(out_of_print: true)
 ```
 
 This will generate SQL like this:
 
 ```sql
-SELECT * FROM clients WHERE (clients.locked = 1)
+SELECT * FROM books WHERE (books.out_of_print = 1)
 ```
 
 The field name can also be a string:
 
 ```ruby
-Client.where('locked' => true)
+Book.where('out_of_print' => true)
 ```
 
 In the case of a belongs_to relationship, an association key can be used to specify the model if an Active Record object is used as the value. This method works with polymorphic relationships as well.
 
 ```ruby
-Article.where(author: author)
-Author.joins(:articles).where(articles: { author: author })
+author = Author.find(3)
+Book.where(author: author)
+Author.joins(:books).where(books: { author: author })
 ```
 
 #### Range Conditions
 
 ```ruby
-Client.where(created_at: (Time.now.midnight - 1.day)..Time.now.midnight)
+Book.where(created_at: (Time.now.midnight - 1.day)..Time.now.midnight)
```

Comment:
Is `1.day.ago.beginning_of_day..1.day.ago.end_of_day` or `1.day.ago.midnight..Time.current.midnight` more readable?

Reply:
I'd argue all of those should be `Date.yesterday.all_day` (which it'd be nice to show off as an example of a non-literal range, but maybe distracting here?)... perhaps there's a more non-standard range we could use? Or switch to an integer field?

Reply:
I didn't know `Date.yesterday.all_day` existed. I love it! I guess it depends on what we assume the reader should know. Can we assume they know what a language level feature is?

If yes, use `all_day`. If no, we could switch to a range on something like `orders_count `.

Reply:
I like your suggestions of more readable and understandable ranges, @gmcgibb.  I think we should stick to a Range since that's what is being talked about.  But I'd like to also include `Date.yesterday.all_day` parenthetically.  _[I didn't know about that either.  thanks @matthewd !]_  So maybe something like:

```
Book.where(created_at: 1.day.ago.midnight..Time.current.midnight)
# Note: The Range could be replaced with Date.yesterday.all_day  
# But this example is meant to explicitly show a Range.
```